packages/core/src/tracing/opentracing/Span.js

```diff
@@ -9,11 +9,66 @@ const opentracing = require('opentracing');
 const spanBuffer = require('../spanBuffer');
 const tracingUtil = require('../tracingUtil');
 
+/**
+ * @typedef {Object} InnerSpanDataSdkCustom
+ * @property {*} tags
+ * @property {*} logs
+ */
+
+/**
+ * @typedef {Object} InnerSpanDataSdk
+ * @property {'local' | 'entry' | 'exit'} type
+ * @property {string} name
+ * @property {InnerSpanDataSdkCustom} custom
+ */
+
+/**
+ * @typedef {Object} InnerSpanData
+ * @property {string} service
+ * @property {InnerSpanDataSdk} sdk
+ */
+
+/**
+ * @typedef {Object} InnerSpan
```

Comment:
*question:* Why is it **Inner**Span. What does the "Inner" refer to?

Reply:
At the time I was confused, as we have more than one definition of a span in different places.
I don't remember exactly why I picked this name. Probably because it is referenced inside another span or some other object.
Basically we have Span class, InstanaSpan, InstanaPseudoSpan, and in many cases we have arbitrary objects with pieces of a span.
Would you suggest a different name? `OpenTracingSpan` maybe?

Reply:
Only saw that now: OpenTracingSpan sounds good but we can also rename later.